src/metabase/search/appdb/scoring.clj

```diff
@@ -145,14 +156,19 @@
 (defn base-scorers
   "The default constituents of the search ranking scores."
   [search-ctx]
+  ;; NOTE: we calculate scores even if the weight is zero, so that it's easy to consider how we could affect any
+  ;; given set of results. At some point, we should optimize away the irrelevant scores for any given context.
   {:text         [:ts_rank :search_vector :query [:inline ts-rank-normalization]]
    :view-count   (view-count-expr search.config/view-count-scaling-percentile)
    :pinned       (truthy :pinned)
    :bookmarked   bookmark-score-expr
    :recency      (inverse-duration [:coalesce :last_viewed_at :model_updated_at] [:now] search.config/stale-time-in-days)
    :user-recency (inverse-duration (user-recency-expr search-ctx) [:now] search.config/stale-time-in-days)
    :dashboard    (size :dashboardcard_count search.config/dashboard-count-ceiling)
-   :model        (model-rank-exp search-ctx)})
+   :model        (model-rank-exp search-ctx)
+   :mine         (equal :search_index.creator_id (:current-user-id search-ctx))
+   :exact        (equal [:lower :search_index.name] [:lower (:search-string search-ctx)])
+   :prefix       (prefix [:lower :search_index.name] (u/lower-case-en (:search-string search-ctx)))})
```

Comment:
we're kind of relying on the fact that `search-string` starts with a name/title in general, right? 😁

Reply:
Since our text ranker also biases towards matches in the name I figured yeah, let's not care about prefixes elsewhere.

We should probably start writing the `table.display_name` field to `search_index.name` column, but I'm not sweating that for now.